colabs/artifact_basics/Artifact_Basics.ipynb

```diff
@@ -0,0 +1,355 @@
+        "id": "u60Orwg5sYuI"
+      },
+      "source": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "# Create a new Artifact version"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "When you need add a new file, you can use the [`artifact.add_file`](https://docs.wandb.ai/ref/python/artifact#add_file) method."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "run = wandb.init(project=\"artifact-basics\")\n",
+        "artifact = run.use_artifact(artifact_or_name=\"my_first_artifact:latest\") # selects the artifact you're adding the file to\n",
+        "artifact.add_file(local_path=\"/content/sample_data/california_housing_test.csv\", name=\"new_file\")\n",
+        "run.log_artifact(artifact)\n",
+        "run.finish()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "This adds a new .csv file called `new_file` to the `my_first_artifact` Artifact. \n",
+        "\n",
+        "Optionally, you can choose to add an entire directory to an Artifact:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "run = wandb.init(project=\"artifact-basics\")\n",
+        "artifact = run.use_artifact(artifact_or_name=\"my_first_artifact:latest\") # selects the artifact you're adding the file to, and makes a new\n",
+        "artifact.add_dir(local_path=\"/content/sample_data\", name=\"new_directory\")\n",
+        "run.log_artifact(artifact, aliases= [\"directory_version\"])\n",
+        "run.finish()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "This adds a new version of `my_first_artifact` with an entire folder with several .csv files. To save your changes, you'll need to [`log`](https://docs.wandb.ai/ref/python/log) or [`save`](https://docs.wandb.ai/ref/python/artifact#save) them. The directory will overwrite the contents of the previous version of the Artifact.\n",
+        "\n",
+        "The Artifact has also been given a custom `aliases`, a label for this Artifact version. While the `alias` is currently `directory_version`, the default aliases are `v1` and `latest `."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "AyANijPlIN5m"
+      },
+      "source": [
+        "# Update an Artifact"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "e4psfYzGeAVS"
+      },
+      "source": [
+        "You can pass new values to update the `description`, `metadata`, and `alias` of an artifact. Call the `save()` method to update the artifact on the W&B servers. You can update an artifact during or outside a W&B Run.\n",
```

Comment:
> update the artifact on the W&B servers

This may be confusing given the nature of artifact storage and different deployment types. Perhaps just 
"update the artifact on Weights & Biases."